data/core/units/monsters/Ant_Egg.cfg

```
@@ -0,0 +1,52 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Giant Ant Egg
+    name= _ "Giant Ant Egg"
+    race=monster
+    image="units/monsters/ant/ant_egg.png"
+    profile="portraits/monsters/ant-egg.webp"
+    # It should just sit there, but default behaviour kicks in and hflips, 
+    # so we make the base frame a blank
+    [standing_anim]
+        start_time=-50
+        [frame]
+            image="misc/blank-hex.png:50"
+        [/frame]
+        [egg_frame]
+            image="units/monsters/ant/ant_egg.png:50"
+            auto_vflip=no
+            auto_hflip=no
+        [/egg_frame]
```

Does adding `auto_flip=no` to the main frame not work?
```suggestion
        [frame]
            image="units/monsters/ant/ant_egg.png:50"
            auto_vflip=no
            auto_hflip=no
        [/frame]
```

---

It works for most of the animation, but there is some sort of gap, whether in attack vs defense animation timing or something else, that ends up with default behaviour blinking in and out, so the egg does some quick flips back & forth.  `[egg_frame]` was the simplest way out.

---

Hmm… feels like there must be something missing here… if we can figure out what it is, I'd definitely prefer not to use an `[egg_frame]`… and there's always the chance it's an actual bug…